docs/website/versioned_docs/version-3.0.0/command-reference/remove-binding.md

```diff
@@ -0,0 +1,18 @@
+---
+title: odo remove binding
+---
+
+The `odo remove binding` command removes the link created between the component and a service via Service Binding.
```

Comment:
Could you please follow the format of this page, with "Description" and "Running the command" section?

https://odo.dev/docs/3.0.0/command-reference/dev